Tests/AppTests/PackageShowModelTests.swift

```diff
@@ -248,7 +248,7 @@ class PackageShowModelTests: SnapshotTestCase {
             defaultBranch: "default",
             name: "Alamofire",
             owner: "Alamofire",
-            readmeUrl: "https://github.com/Alamofire/Alamofire/master/README.md"
+            readmeUrl: "https://raw.githubusercontent.com/foo/bar/main/README.md"
```

Comment:
As part of this PR, I made the test data we're using for the readmeUrl a little more real-world. It was a little confusing for the test URLs to be to HTML representations of the markdown when in actuality we're getting raw Markdown.